test/d_do_test.d

```diff
@@ -668,7 +669,7 @@ int tryMain(string[] args)
                 }
                 else version (linux)
                 {
-                    auto script = test_app_dmd_base ~ to!string(i) ~ ".gdb";
+                    auto script = test_app_dmd_base ~ to!string(permuteIndex) ~ ".gdb";
```

Comment:
You need to update the `version(Windows)` block above too.

Reply:
done